src/convertToObject.js

```diff
@@ -7,6 +7,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const splitString = sourceString
+    .split(';')
+    .filter((string) => string.includes(':'));
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable.